client/src/components/App.jsx

```diff
@@ -95,8 +95,11 @@ class App extends React.Component {
           <GlobalStyles/>
           <Container>
             <h1>Welcome to Atelier!</h1>
-            <Overview currentProductId={this.state.currentProductId}></Overview>
-            <div><Reviews currentProductId = '40344'currentProductRating = {this.state.currentProductRating}/></div>
```

Comment:
Did you delete the Reviews component?

Reply:
Shouldn't, I only commented it out.

Reply:
Yeah, it's in below section.

Reply:
ah gotcha, I added it back in when I merged with mine!